hsigmoid/network_conf.py

```diff
@@ -30,18 +31,27 @@ def ngram_lm(hidden_size, embed_size, dict_size, gram_num=4, is_train=True):
             initial_std=1. / math.sqrt(embed_size * 8), learning_rate=1))
 
     if is_train == True:
-        cost = paddle.layer.hsigmoid(
+        return paddle.layer.hsigmoid(
             input=hidden_layer,
             label=target_word,
-            num_classes=dict_size,
```

Comment:
According to the code: https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/gserver/layers/HierarchicalSigmoidLayer.cpp#L27, maybe `num_classes` is necessary?

Reply:
Done.